test/e2e/apimachinery/webhook.go

```diff
@@ -154,10 +157,12 @@ var _ = SIGDescribe("AdmissionWebhook", func() {
 		testMutatingPodWebhook(f)
 	})
 
-	It("Should not be able to prevent deleting validating-webhook-configurations or mutating-webhook-configurations", func() {
-		webhookCleanup := registerWebhookForWebhookConfigurations(f, context)
-		defer webhookCleanup()
-		testWebhookForWebhookConfigurations(f)
+	It("Should not be able to prevent deleting or mutate validating-webhook-configurations or mutating-webhook-configurations", func() {
```

Comment:
minor nit: the wording here is a little confusing. I'm not sure if "deleting or mutate" binds to "validating-webhook-configurations" only (in which case mutate is maybe a typo)? or whether there are three conditions "prevent deleting, mutate validating-..., [verb]? mutating-..." or something else

Reply:
Good point, the wording is probably too confusing here. I had intended it to mean that all of the following are not possible to implement via a dynamic admission webhook:
* prevent deleting validating-webhook-configurations
* prevent deleting mutating-webhook-configurations
* mutate validating-webhook-configurations
* mutate mutating-webhook-configurations

Reply:
Do you think it would be simpler to say something like:
"Should not be able to mutate or prevent deleting webhook configuration objects"?

Reply:
yeah, that sounds better!